### ISSUE TITLE
'Console Command Definition is empty' error

### ISSUE THREAD

**Issue by CraigHarley** [llaski/nova-scheduled-jobs#2]
I've added a few commands to the schedule and I'm getting this error when I open the tool. 

Have I done something wrong?

`app/Kernal.php`
```
    protected function schedule(Schedule $schedule)
    {
        $schedule->command('backup:clean')->daily()->at('01:00');
        $schedule->command('backup:run')->daily()->at('02:00');
    }
```

**Comment by llaski** [comment]
@CraigHarley Not really sure what error your getting - can you post it?

Also the location of your scheduled commands should be in app/Console/Kernel.php.

**Comment by CraigHarley** [comment]
That was a typo, the scheduled commands are in `app/Console/Kernel.php`

Here's the error message:
![image](https://user-images.githubusercontent.com/8432365/45144251-8302dd00-b1b5-11e8-9cb1-abe08b5ffeb6.png)
![image](https://user-images.githubusercontent.com/8432365/45144384-d83eee80-b1b5-11e8-9241-451243e7f8ca.png)

The commands in question run from the console when ran manually.

**Comment by llaski** [comment]
@CraigHarley Hm very weird - I added the commands to my local setup and it worked with no issue. My only thought is that the format of the commands returned from the events method from the scheduler is not fitting the regex in `CommandEvent` class. Would you be able to go into `Llaski\NovaScheduledJobs\Http\Controllers\JobsController` and throw a `dd($schedule->events())` on line 21 before the return statement and post the output here?

**Comment by CraigHarley** [comment]
@llaski here you go:

![image](https://user-images.githubusercontent.com/8432365/45209065-f6732000-b283-11e8-94b4-ab0da09fcc16.png)

**Comment by llaski** [comment]
@CraigHarley Thanks for sending that - believe I fixed the issue and just released version 1.3.0, so give that a whirl and let me know if it works now.

**Comment by CraigHarley** [comment]
I can confirm that's working, thanks @llaski
